client/src/Utils/auxiliaryFunctions/auxiliaryFunctions.ts

```diff
@@ -9,16 +9,20 @@ const get = (obj: any, path: string, defaultValue = undefined) => {
 
 const isIdValid = (id: string | null | undefined) => {
   let sum = 0;
-  if (id?.length === 9) {
-    Array.from(id)?.forEach((digit: string, index: number) => {
-      let digitMul = parseInt(digit) * ((index % 2) + 1);
-      if (digitMul > 9) {
-        digitMul -= 9;
-      }
-      sum += digitMul;
-    })
+  if(Boolean(id)) {
```

Comment:
If you check ```Boolean(id)``` I don't think you need the ?

Reply:
fixed